lib/accounting/test/create_card.rb

```diff
@@ -0,0 +1,53 @@
+module Accounting
```

Comment:
This seems to be just to make the tests pass.  Do we need to refactor tests too to reflect the change to Accept.js?

Reply:
In current Accounting gem we use Accept.js payment nonce but I don't know a way to create payment nonce in RSpec. But we need to create Accounting::Payment in specs to run Service and Hook specs. This class provides the way to create Accounting::Payment without using Accept.js payment nonce but using Payment Card informations.

Reply:
I think I have done all possible refactoring on Specs at this point.